extensions/blocks/opentable/restaurant-picker.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, FormTokenField } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __, _n } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useRestaurantSearch from './use-restaurant-search';
+
+const MAX_SUGGESTIONS = 20;
+const embedRegex = /<script type=\'text\/javascript\' src=\'\/\/www.opentable.com\/widget\/reservation\/loader\?[^']+\'><\/script>/;
```

Comment:
It looks like OpenTable has country specific domain names for the embeds, eg. https://www.opentable.com.au/widget/reservation/preview?rid=122582&lang=en-US has www.opentable.com.au, so this regex needs to be loosened up a bit.

Reply:
I did wonder about that. We might need to check how we're building the URL embed too

Reply:
+1
It also has `.co.uk` and `.it` (just two I've randomly checked).

The TLD also determines the `domain` parameter of the widget, which as far as I can see, it's just the TLD without dots.
E.g.
`.com` -> `domain=com`
`.co.uk` -> `domain=couk`

Reply:
Done in de41f11b473d6e39906a27b7632e5d2a2c00c2de